config/initializers/logger.rb

```diff
@@ -0,0 +1,13 @@
+# suppresses routing errors
+if Rails.env.production?
+  class ActionDispatch::DebugExceptions
+    alias_method :old_log_error, :log_error
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Style/Alias: Use `alias` instead of `alias_method` in a class body.